### PR TITLE
chore: VScode formatter settings

### LINKED ISSUE #449: add vscode settings to use correct formatting settings
I use vscode and IDE and figured out that some changes will be done by plugins e.g. prettier to formatting the code.
I add a settings file to ignore this and use eslint options instead.

### PR DESCRIPTION
**Pull request by pharindoko**
closes #449

### CODE REVIEW

**Comment by kichik** [comment]
Why disable Svelte formatting?

File: .projenrc.js [hunk 1]
```diff
@@ -150,4 +150,15 @@ cdkConfig.json.addDeletionOverride('output');
...
 project.eslint.allowDevDeps('src/lambda-github.ts');
 
+project.vscode.settings.addSettings({
+  'editor.formatOnSave': true,
+  'editor.codeActionsOnSave': {
+    'source.fixAll.eslint': true,
+  },
+  'eslint.format.enable': true,
+  'prettier.enable': false,
+  'svelte.plugin.svelte.format.enable': false,
```
Comment (kichik) [inline .projenrc.js:161]: Why no Svelte formatting?

**Comment by pharindoko** [comment]
Hey the vscode extension for svelte would overrule the other formatters and add double quotes and remove the indenting for the svelte files. It would chnage each and every line :)

**Comment by kichik** [comment]
> Hey the vscode extension for svelte would overrule the other formatters and add double quotes and remove the indenting for the svelte files. It would change each and every line in the svelte file

Oh, it's not the eslint plugin of Svelte. It's VScode. Got it.

Thanks.

**Review by kichik — APPROVED** [review]

**Comment by pharindoko** [comment]
> > Hey the vscode extension for svelte would overrule the other formatters and add double quotes and remove the indenting for the svelte files. It would change each and every line in the svelte file
> 
> 
> 
> Oh, it's not the eslint plugin of Svelte. It's VScode. Got it.
> 
> 
> 
> Thanks.
Yes 
And it's the first thing svelte users will install as extension  and it is proposed by vscode itself as far as I remember...

File: .projenrc.js [hunk 2]
```diff
@@ -150,4 +150,16 @@ cdkConfig.json.addDeletionOverride('output');
 project.eslint.allowDevDeps('src/lambda-helpers.ts');
 project.eslint.allowDevDeps('src/lambda-github.ts');
 
+// vscode auto formatting
+project.vscode.settings.addSettings({
```
Comment (pharindoko) [inline .projenrc.js:154]: Ah you added it :) thanks...
Wanted to do the same now...

**Pull request merged**